### PR TITLE
Gengo

### PR DESCRIPTION
**Pull request by lavalamp**
go2idl is now gengo.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31731)

### CODE REVIEW

File: cmd/libs/go2idl/deepcopy-gen/main.go [hunk 1]
```diff
@@ -48,8 +48,10 @@ limitations under the License.
 package main
 
 import (
-	"k8s.io/kubernetes/cmd/libs/go2idl/args"
-	"k8s.io/kubernetes/cmd/libs/go2idl/deepcopy-gen/generators"
+	"path/filepath"
+
+	"k8s.io/gengo/args"
+	"k8s.io/gengo/examples/deepcopy-gen/generators"
```
Comment (thockin) [inline cmd/libs/go2idl/deepcopy-gen/main.go]: This is a little clunky, but I'll allow it :)

**Comment by lavalamp** [comment]
OK, propagated changes & rebased.

**Comment by lavalamp** [comment]
argh. I hate godep.

**Comment by lavalamp** [comment]
OK. I think with this change all the verification scripts will be happy.

**Comment by lavalamp** [comment]
Er, I mean, now that I've rebased it, it should be good.

**Comment by lavalamp** [comment]
Yes, everything passed! this was basically a rebase so I'm going to claim I inherit the previous lgtm :)

**Pull request merged**